specification/communication/data-plane/CallingServer/preview/2022-04-07-preview/communicationservicescallingserver.json

```diff
@@ -891,6 +932,62 @@
         "callbackUri": {
           "description": "The callback uri.",
           "type": "string"
+        },
+        "mediaStreamingConfiguration": {
+          "$ref": "#/definitions/MediaStreamingConfigurationDto",
+          "description": "Media Streaming Configuration."
+        }
+      }
+    },
+    "MediaStreamingTransportTypeDto": {
+      "description": "The type of tranport to be used for media streaming, eg. Websocket",
+      "enum": [
+        "websocket"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "MediaStreamingTransportTypeDto",
+        "modelAsString": true
+      }
+    },
+    "MediaStreamingContentTypeDto": {
+      "description": "Content type to stream, eg. audio, audio/video",
+      "enum": [
+        "audio"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "MediaStreamingContentTypeDto",
+        "modelAsString": true
+      }
+    },
+    "MediaStreamingAudioChannelTypeDto": {
+      "description": "Audio channel type to stream, eg. unmixed audio, mixed audio",
+      "enum": [
+        "mixed",
+        "unmixed"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "MediaStreamingAudioChannelTypeDto",
+        "modelAsString": true
+      }
+    },
+    "MediaStreamingConfigurationDto": {
```

Comment:
We typically don't see "Dto" in entity names. We can probably just drop the suffix from all these types and make them e.g.:

* `MediaStreamingConfiguration`

* `MediaStreamingAudioChannelType`

* etc